src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginManager.cs

```diff
@@ -94,9 +94,6 @@ public static T CreateTestExtension<T>(Type extensionType)
     /// <typeparam name="TMetadata">
     /// Type of metadata of required extensions
     /// </typeparam>
-    /// <typeparam name="TMetadata">
```

Comment:
All these changes seem wrong, `TMetadata` type parameter does exist.

Reply:
I just spotted that there was a typo in the `IMetadata`

Reply:
I think `TMetadata` is correct, it's just that `IMetadata` is missing (signature is `<TPluginInfo, TExtension, IMetadata, TMetadata>`).

Reply:
Right, that the fix I've applied in the final interation.